components/GameScreen.vue

```diff
@@ -436,6 +428,11 @@ watch(gameState, async (newState, prevState) => {
       class="flex flex-col shadow ml-2"
       :style="{ maxWidth: gameState?.width + 'px' }"
     >
+      <div class="flex flex-row justify-end mb-2 mt-1 mx-4 gap-6">
+        <ButtonLink @click="toggleFollowMeMode">
+          {{ isFollowing ? "stop following my bot" : "follow my bot" }}
+        </ButtonLink>
+      </div>
```

Comment:
We've just merged a [change](https://github.com/move-fast-and-break-things/aibyss/commit/1e0f17d93031b570d60880fb56275957d83dc2d3) that makes the `GameScreen` visible without login. Let's hide the "follow my bot" button if the user isn't logged in.